src/postgres/src/test/regress/expected/yb_pg_partition_prune.out

```diff
@@ -0,0 +1,2854 @@
+--
```

Comment:
the .sql file for this does not seem to be here.

Reply:
Added it in the next patch, really sorry about missing it.